### PR TITLE
add Vue support

### PR DESCRIPTION
**Pull request by riophae**
Vue SFCs look like this:

vue-component

Actually it's just a special HTML. See: https://vuejs.org/v2/guide/single-file-components.html

### CODE REVIEW

File: components.json [hunk 1]
```diff
@@ -48,12 +48,13 @@
...
 			"title": "Markup",
-			"alias": ["html", "xml", "svg", "mathml"],
+			"alias": ["html", "xml", "svg", "mathml", "vue"],
 			"aliasTitles": {
 				"html": "HTML",
 				"xml": "XML",
 				"svg": "SVG",
-				"mathml": "MathML"
+				"mathml": "MathML",
+				"vue": "Vue",
```
Comment (mAAdhaTTah) [inline components.json]: The comma at the end of this line is invalid JSON. That's why the build is failing.

Comment (riophae) [reply components.json]: Fixed.

**Comment by Golmote** [comment]
I'm not sure I'm in favor of this addition. Vue is not a language is it? It does not even have a special syntax?

What issue is this PR supposed to fix? It it to be able to add class `language-vue` to code blocks?

**Comment by riophae** [comment]
@Golmote Yeah... Sort of like that.

**Comment by Golmote** [comment]
@mAAdhaTTah I'm not worried about the filesize, we have minified versions to deal with this (obtw it appears to be missing from this PR). I'm worried about the semantics...

I went through the VueJS doc, and I see it has a Mustache-like template syntax. If we were to add a component for this (maybe it could handle multiple templating languages depending on how similar they are), we could then consider it a "language". But just adding `vue` as a alias for `markup` seems not appropriate to me.

**Comment by riophae** [comment]
@Golmote I agree with you.

**Comment by riophae** [comment]
For example, the current `markup` definition doesn't handle this well:

```
<div>{{ look_at_me }}</div>
```

**Pull request closed without merging**